docs/markdown/40-reuse/00-reusable-workflow.md

```diff
@@ -0,0 +1,91 @@
+<!-- .slide: -->
+
+# Reusable Workflows
+
+The same thinking can be apply when creating reusable workflows.
+We are creating them to extract orchestration patterns to be usable to others like we extract tasks to be usable to others when creating actions.
+
+The only type of action comparable to reusable workflows is the **composite** one.
+What the difference between them?
+
+##--##
+
+# Reusable Workflows
+
+## Example Workflow
+
+```yaml
+on:
+  workflow_call:
+    inputs:
+      workflow_input:
+        description: 'workflow_input'
+        required: true
+        type: string
+      workflow_env:
+        description: 'workflow_env'
+        required: true
+        type: string
+    secrets:
+      workflow_secret:
+        description: 'workflow_secret'
+        required: true
+    outputs:
+      workflow_output:
+        description: 'workflow_output'
+        value: ${{ jobs.workflow_job.outputs.job_output }}
+jobs:
+  workflow_job:
+    runs-on: ubuntu-latest
+    outputs:
+      job_output: ${{ steps.step1.outputs.action_output }}
+    steps:
+      - id: step1
+        uses: some/action@v1
+        with:
+          action_input1: ${{ inputs.workflow_input }}
+          action_input2: ${{ secrets.workflow_secret }}
+          env:
+            ACTION_ENV: ${{ inputs.workflow_env }}
+```
+
+##--##
+
+# Reusable Workflows
+
+## Example Action
+
+```yaml
+inputs:
+  action_input:
+    description: 'action_input'
+    required: true
+  action_secret:
+    description: 'action_secret'
+    required: true
+outputs:
+  action_output:
+    description: 'action_output'
+    value: ${{ jobs.workflow_job.outputs.job_output }}
+runs:
+  using: composite
+  steps:
+    - id: step1
+      uses: some/action@v1
+      with:
+        action_input1: ${{ inputs.action_input }}
+        action_input2: ${{ inputs.action_secret }}
+      env:
+        ACTION_ENV: ${{ env.WORKFLOW_ENV }}
+```
+
+##--##
+
+# TODO Reusable Workflows
+
+- Only actions can be call from both
```

Comment:
Some changes about that
- https://github.blog/changelog/2022-08-22-github-actions-improvements-to-reusable-workflows-2/

Reply:
> You can connect up to four levels of workflows.